Sources/SwiftDocC/Model/SourceLanguage.swift

```diff
@@ -149,4 +149,31 @@ public struct SourceLanguage: Hashable, Codable {
     
     /// The list of programming languages that are known to DocC.
     public static var knownLanguages: [SourceLanguage] = [.swift, .objectiveC, .javaScript, .data, .metal]
+    
+    enum CodingKeys: CodingKey {
+        case name
+        case id
+        case idAliases
+        case linkDisambiguationID
+    }
+    
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: SourceLanguage.CodingKeys.self)
+        
+        let name = try container.decode(String.self, forKey: SourceLanguage.CodingKeys.name)
+        let id = try container.decode(String.self, forKey: SourceLanguage.CodingKeys.id)
+        let idAliases = try container.decodeIfPresent([String].self, forKey: SourceLanguage.CodingKeys.idAliases) ?? []
```

Comment:
I explicitly implemented the Codable conformance here to match the spec where `idAliases` and `linkDisambiguationID` are optional.